src/main/java/Parser.java

```diff
@@ -0,0 +1,62 @@
+public class Parser {
+
+    public Parser() {
+    }
+
+    public static void parse(String command, TaskList tasks, Storage storage) {
```

Comment:
Perhaps use a switch here instead of multiple if and else if statements? It would make the code more readable. 😄